fe/fe-core/src/main/java/org/apache/doris/load/StreamLoadRecordMgr.java

```diff
@@ -0,0 +1,112 @@
+                        client = ClientPool.backendPool.borrowObject(address);
+                        TStreamLoadRecordResult result = client.getStreamLoadRecord(backend.getLastStreamLoadTime());
+                        Map<String, TStreamLoadRecord> streamLoadRecordBatch = result.getStreamLoadRecord();
+                        LOG.info("receive stream load audit info from backend: {}. batch size: {}", backend.getHost(), streamLoadRecordBatch.size());
+                        for (Map.Entry<String, TStreamLoadRecord> entry : streamLoadRecordBatch.entrySet()) {
+                            TStreamLoadRecord streamLoadItem= entry.getValue();
+                            LOG.info("receive stream load record info from backend: {}. label: {}, db: {}, tbl: {}, user: {}, user_ip: {}," +
+                                            " status: {}, message: {}, error_url: {}, total_rows: {}, loaded_rows: {}, filtered_rows: {}," +
+                                            " unselected_rows: {}, load_bytes: {}, start_time: {}, finish_time: {}.",
+                                    backend.getHost(), streamLoadItem.getLabel(), streamLoadItem.getDb(), streamLoadItem.getTbl(), streamLoadItem.getUser(), streamLoadItem.getUserIp(),
+                                    streamLoadItem.getStatus(), streamLoadItem.getMessage(), streamLoadItem.getUrl(), streamLoadItem.getTotalRows(), streamLoadItem.getLoadedRows(),
+                                    streamLoadItem.getFilteredRows(), streamLoadItem.getUnselectedRows(), streamLoadItem.getLoadBytes(), streamLoadItem.getStartTime(),
+                                    streamLoadItem.getFinishTime());
+
+                            AuditEvent auditEvent = new StreamLoadAuditEvent.AuditEventBuilder().setEventType(EventType.STREAM_LOAD_FINISH)
+                                    .setLabel(streamLoadItem.getLabel()).setDb(streamLoadItem.getDb()).setTable(streamLoadItem.getTbl())
+                                    .setUser(streamLoadItem.getUser()).setClientIp(streamLoadItem.getUserIp()).setStatus(streamLoadItem.getStatus())
+                                    .setMessage(streamLoadItem.getMessage()).setUrl(streamLoadItem.getUrl()).setTotalRows(streamLoadItem.getTotalRows())
+                                    .setLoadedRows( streamLoadItem.getLoadedRows()).setFilteredRows(streamLoadItem.getFilteredRows())
+                                    .setUnselectedRows(streamLoadItem.getUnselectedRows()).setLoadBytes(streamLoadItem.getLoadBytes())
+                                    .setStartTime(streamLoadItem.getStartTime()).setFinishTime(streamLoadItem.getFinishTime())
+                                    .build();
+                            Catalog.getCurrentCatalog().getAuditEventProcessor().handleAuditEvent(auditEvent);
+                            if (entry.getKey().compareTo(backend.getLastStreamLoadTime()) > 0) {
+                                backend.setLastStreamLoadTime(entry.getKey());
```

Comment:
We can find the max timestamp of this batch and call `setLastStreamLoadTime` once.

Reply:
> We can find the max timestamp of this batch and call `setLastStreamLoadTime` once.

It is reasonable.